html2asketch/model/text.js

```diff
@@ -2,8 +2,8 @@ import Base from './base';
 import {RESIZING_CONSTRAINTS} from '../helpers/utils';
 
 class Text extends Base {
-  constructor({x, y, width, height, text, style, multiline}) {
-    super();
+  constructor({x, y, width, height, text, style, multiline, id}) {
+    super({id});
```

Comment:
please do the same for all classes that extend Base (e.g. SymbolMaster, SymbolInstence, …)